javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,55 @@
+const url = 'https://open.er-api.com/v6/latest/USD';
+const amountInput = document.getElementById('amount');
+const fromSelect = document.getElementById('from');
+const toSelect = document.getElementById('to');
+const result = document.getElementById('result');
+let exchangeRates, currencies;
+
+async function fetchExchangeRates() {
+  const response = await fetch(url);
```

Comment:
Nicely done, but would be better with some error handling in case something goes wrong.